src/RepairManager/Rules/ecc_rule.py

```diff
@@ -101,8 +100,6 @@ def take_action(self):
                 else:
                     body += f'{node_name}: Successfully marked as unschedulable\n'
 
-        if not all_nodes_already_unscheduled:
```

Comment:
Looks like `all_nodes_already_unscheduled` is no longer used. Consider deleting?